src/riak_kv_backend.erl

```diff
@@ -31,33 +30,52 @@
 -type fold_objects_fun() :: fun((binary(), binary(), term(), any()) ->
                                        any() |
                                        no_return()).
+
```

Comment:
What do the `Index` and `SecondaryIndex` mean in this type spec? I could not find any information about them here http://erlang.org/documentation/doc-5.10.3/doc/reference_manual/typespec.html